.azure-pipelines/templates/run-tests.yml

```diff
@@ -7,19 +7,19 @@ parameters:
 
 steps:
 - ${{ if eq(parameters.test, 'true') }}:
-  - script: ddev test --cov --junit ${{ parameters.check }}
+  - script: ddev --${{ parameters.repo }} test --cov --junit ${{ parameters.check }}
```

Comment:
Not sure this is needed since we are already setting the repo here: https://github.com/DataDog/integrations-core/blob/b6c991b2cc953a71a91de06382c27a766ce8b856/.azure-pipelines/templates/install-deps.yml#L32-L33

Reply:
That is already happening on a prior step